cmd/client/errors.go

```diff
@@ -5,10 +5,9 @@ import (
 )
 
 var (
-	errNoMultiaddr      = errors.New("must provide peer's multiaddress with --multiaddr")
 	errNoMinAmount      = errors.New("must provide non-zero --min-amount")
 	errNoMaxAmount      = errors.New("must provide non-zero --max-amount")
 	errNoExchangeRate   = errors.New("must provide non-zero --exchange-rate")
-	errNoOfferID        = errors.New("must provide --offer-id")
```

Comment:
The `--offer-id` is now marked as required by `urfave`, so this error wasn't used anymore.